configure.ac

```diff
@@ -3100,6 +3100,12 @@ then
         AM_CFLAGS="$AM_CFLAGS -DALT_ECC_SIZE"
     fi
 
+    if test "$ENABLED_OPENSSLEXTRA" = "yes"
+    then
+        # Checking point is on curve requires this.
+        AM_CFLAGS="$AM_CFLAGS -DUSE_ECC_B_PARAM"
```

Comment:
You might consider moving this force enable of `USE_ECC_B_PARAM` with `OPENSSL_EXTRA` into settings.h?

Then you could also force it off for the complex macro logic:

```
!defined(WOLFSSL_ATECC508A) && \
     !defined(WOLFSSL_ATECC608A) && !defined(WOLFSSL_CRYPTOCELL) && \
     !defined(WOLFSSL_SE050) && !defined(WOLF_CRYPTO_CB_ONLY_ECC) && \
     !defined(WOLFSSL_STM32_PKA)
```

Reply:
Change made as suggested.